surf/apps/communities/migrations/0008_auto_20191021_0856.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 2.0.13 on 2019-10-21 08:56
```

Comment:
On the test server. Reverse all migrations. Then delete all migrations. Then 
python manage.py makemigrations -n new_community_models
Create a new data dump. Locally delete your database. Import new dump.
Should create a cleaner migration history

Reply:
Squashed the migrations where possible.